### ISSUE TITLE
Missing dependency Apache Commons Lang3

### ISSUE THREAD

**Issue by zosocanuck** [jenkinsci/venafi-vcert-plugin#1]
### Description

After installing the Venafi Machine Identity Protection plugin for Jenkins and running a sample pipeline, the following error occurs:

```
Running in Durability level: MAX_SURVIVABILITY
[Pipeline] Start of Pipeline
[Pipeline] node
Running on Jenkins in C:\Users\jenkins\AppData\Local\Jenkins.jenkins\workspace\test
[Pipeline] {
[Pipeline] venafiVcertRequestCertificate
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
java.lang.NoClassDefFoundError: org/apache/commons/lang3/StringUtils
	at com.venafi.vcert.sdk.VCertClient.<init>(VCertClient.java:32)
	at io.jenkins.plugins.venafivcert.CertRequestBuilder.createClient(CertRequestBuilder.java:315)
	at io.jenkins.plugins.venafivcert.CertRequestBuilder.perform(CertRequestBuilder.java:215)
	at jenkins.tasks.SimpleBuildStep.perform(SimpleBuildStep.java:112)
Caused: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:92)
	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:70)
	at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Finished: FAILURE
```

### Steps to Reproduce

1. Install Venafi plugin for Jenkins
2. Run sample pipeline per documentation
3. [and so on...]

**Expected behavior:** Should work and certificate, private key, and chain files should be produced.

**Actual behavior:** Missing dependency error occurs per above.

### Environment

**Jenkins-Version:** 2.249.3 on Windows 10
**Java-Version:** JDK 1.8.0_271

**Plugin-Version:** 1.0.0
**Master/Slave Setup:** No

**Comment by zosocanuck** [comment]
Workaround is to install Apache commons lang4 jar file as part of Java distribution.

**Comment by FooBarWidget** [comment]
Issue reproduced.

**Comment by FooBarWidget** [comment]
Fix released in 1.0.1.
